konstellation/assetlist.json

```diff
@@ -0,0 +1,23 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_id": "darchub",
+    "assets": [
+        {
+            "description": "The native token of Konstellation Network",
+            "denom_units": [
+                {
+                    "denom": "udarc",
+                    "exponent": 0
+                },
+                {
+                    "denom": "darc",
+                    "exponent": 6
+                }
+            ],
+            "base": "darc",
```

Comment:
Are you sure this shouldn't be udarc?

Reply:
Yep, It should be :) Thank you. I'll fix it.